logical/framework/field_data_test.go

```diff
@@ -467,6 +468,40 @@ func TestFieldDataGet(t *testing.T) {
 			},
 		},
 
+		"type header, keypair string array": {
+			map[string]*FieldSchema{
+				"foo": {Type: TypeHeader},
+			},
+			map[string]interface{}{
+				"foo": []interface{}{"key1=value1", "key2=value2", "key3=1"},
```

Comment:
What happens if a header key name is repeated?  I would expect it to end up with a list of values but I don't see a test for that.

Reply:
Additionally, if you provide a single value, I believe it will come in as a string interface and not a list.  See TypeCommaStringSlice.

Reply:
Thanks! I'll check out TypeCommaStringSlice.

On the double-key thing, I didn't add a test case for that because of the format of the test cases being a map, which obviously doesn't allow the same key multiple times. I'll break out a separate test for that.

Reply:
But I would expect CLI params of `key1=value1 key1=value2` to result in:

```map[string]interface{}{
    "key1": []interface{}{"value1", "value2"}
}
```

... or something like that :).

Reply:
Oh I see what you're saying. :+1: